talk/talk-api.yaml

```
@@ -2536,26 +2588,44 @@ components:
           items:
             $ref: '#/components/schemas/CallParticipant'
         direction:
-          type: string
+          $ref: '#/components/schemas/CallDirection'
         status:
-          type: string
+          $ref: '#/components/schemas/CallStatus'
         recording:
           $ref: '#/components/schemas/Recording'
-        token:
-          type: string
-          description: Twilio capability token
         record:
           type: boolean
           description: If true this call is being recorded now
         createdAt:
           type: integer
           format: int64
           description: Timestamp of the create
-      required:
-        - id
-        - caller
-        - participants
-        - createdAt
+    CallStatus:
+      description: Status of the call
+      type: string
+      enum:
+        - IN_PROGRESS
+        - FINISHED
```

```suggestion
        - ENDED
```
I think in terms of calls "end" is the terminology typically used. It's a minor detail though and does not actually make any difference.

---

I think `finished` sound a bit more natural :D